packages/expo-mail-composer/plugin/src/withMailComposer.ts

```diff
@@ -32,11 +32,13 @@ const mailClientURLs: string[] = [
 
 const withMailComposer: ConfigPlugin = (config) => {
   return withInfoPlist(config, (config) => {
-    config.modResults.LSApplicationQueriesSchemes = [
-      ...(config.modResults.LSApplicationQueriesSchemes ?? []),
-      ...mailClientURLs,
+    const existingSchemes = config.modResults.LSApplicationQueriesSchemes ?? [];
+    const existingSet = new Set(existingSchemes);
+    const newSchemes = [
+      ...existingSchemes,
+      ...mailClientURLs.filter((scheme) => !existingSet.has(scheme)),
     ];
```

Comment:
```suggestion
    const newSchemes = [...new Set([...existingSchemes ,...mailClientURLs])]
```

feels a little more straightforward

Reply:
Absolutely, I have updated the code